gut_to_soil_manuscript_figures/_methods.py

```diff
@@ -44,12 +44,11 @@ def pcoa_2d(output_dir: str, metadata: qiime2.Metadata,
     swap_axes = str(swap_axes)
     himalaya = str(himalaya)
     pit_toilet = str(pit_toilet)
-    export_legend = str(export_legend)
+    export_legend_str = str(export_legend)
 
     plot_fp = os.path.join(output_dir, 'pcoa_plot.png')
 
-    if export_legend:
-        legend_fp = os.path.join(output_dir, 'legend.png')
+    legend_fp = os.path.join(output_dir, 'legend.png')
```

Comment:
```suggestion
```
This doesn't need to get assigned unless `export_legend` is set to True - so I'm going to move it down to the branch that also adds the legend as a separate item in the index.html page.